### PR TITLE
Feature/sof 6782

### PR DESCRIPTION
**Pull request by VsevolodX**
Edits to make dropdown look correctly in Materials Designer

### CODE REVIEW

File: src/mui/components/nested-dropdown/NestedDropdownItem.tsx [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
 import ListItemIcon from "@mui/material/ListItemIcon";
 import MenuItem from "@mui/material/MenuItem";
+import { styled } from "@mui/material/styles";
 import Typography, { TypographyProps } from "@mui/material/Typography";
 import React, { useCallback } from "react";
 
+const StyledListItemIcon = styled(ListItemIcon)({
+    marginLeft: "10px", // this is ridiculus, but the only way to align icons
+    marginRight: "-15px", // the only solution for now
+});
```
Comment (zoomchik) [inline src/mui/components/nested-dropdown/NestedDropdownItem.tsx:11]: Try to make it absolute positioned

Comment (VsevolodX) [reply src/mui/components/nested-dropdown/NestedDropdownItem.tsx:11]: There's a default right margin of some set size for ItemListIcon, I couldn't find a way to adjust or access it. Only an override like this. 
With absolute position it still would have a margin. 

original:
image

styled with margins:
image

Comment (VsevolodX) [reply src/mui/components/nested-dropdown/NestedDropdownItem.tsx:11]: With the `position: absolute, right: 0` there's still the default margin and whole idea of flow positioning of elements in the menu list breaks. 
image

**Review by azech-hqs — APPROVED** [review]
looks good 👍

**Pull request merged**